src/structures/Team.js

```diff
@@ -58,7 +58,7 @@ class Team extends Base {
    * @readonly
    */
   get owner() {
-    return this.members.get(this.ownerID) || null;
+    return this.members.cache.get(this.ownerID) || null;
```

Comment:
```suggestion
    return this.members.get(this.ownerID) || null;
```
Team#members is a collection, see [line 47](https://github.com/RDambrosio016/discord.js/blob/remove-datastores/src/structures/Team.js#L47), not a manager